### PR TITLE
React layout partial cleaning

### PR DESCRIPTION
**Pull request by Gnito**
- Flattened _head.haml & _marketplace_head.haml into the layout page
- Started to remove unnecessary stuff
- Javascript disabled is styled in base.css

### CODE REVIEW

File: app/views/layouts/react_page.haml [hunk 1]
```diff
@@ -1,4 +1,107 @@
-= render :partial => 'layouts/head'
+!!! 5
```
Comment (Gnito) [inline app/views/layouts/react_page.haml]: I have no idea what this is

Comment (sktoiva) [inline app/views/layouts/react_page.haml]: Produces:

```html
XHTML 5
<!DOCTYPE html>
```

Comment (sktoiva) [inline app/views/layouts/react_page.haml]: Ref: http://haml.info/docs/yardoc/file.REFERENCE.html#doctype_

File: app/assets/stylesheets/react_page/styles.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+//= require ./normalize
+//= require ./base
```
Comment (sktoiva) [inline app/assets/stylesheets/react_page/styles.scss]: Missing newline at EOF.

File: app/assets/stylesheets/react_page/normalize.css [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+
+/**
+ * 1. Correct the inability to style clickable types in iOS and Safari.
+ * 2. Change font properties to `inherit` in Safari.
+ */
+
+::-webkit-file-upload-button {
+  -webkit-appearance: button; /* 1 */
+  font: inherit; /* 2 */
+}
```
Comment (sktoiva) [inline app/assets/stylesheets/react_page/normalize.css]: Missing newline at EOF

File: app/views/layouts/react_page.haml [hunk 4]
```diff
@@ -1,4 +1,108 @@
...
+
+%head
+  %meta{charset: "utf-8"}
+
+  -# Kissmetrics is deprecated, use Google Tag Manager instead
+  = render :partial => "layouts/kissmetrics"
```
Comment (sktoiva) [inline app/views/layouts/react_page.haml]: Who knows about this? Is this needed? The comment says it is deprecated and GTM should be used instead and Tag Manager is used on the next line.

File: app/views/layouts/react_page.haml [hunk 5]
```diff
@@ -1,4 +1,108 @@
...
+  = render :partial => "layouts/google_analytics_script", locals: {feature_flags: FeatureFlagHelper.feature_flags, community_id: Maybe(@current_community).id.or_else, google_analytics_key: Maybe(@current_community).google_analytics_key.or_else}
+
+  - font_path = APP_CONFIG[:font_proximanovasoft_url]
+  - if font_path.present? && FeatureFlagHelper.feature_enabled?(:topbar_v1)
```
Comment (sktoiva) [inline app/views/layouts/react_page.haml]: I guess we shouldn't come to this page at all without topbar enabled?

File: app/assets/stylesheets/react_page/styles.scss (hunk 2 above)
Comment (Gnito) [inline app/assets/stylesheets/react_page/styles.scss]: Classic :)

File: app/views/layouts/react_page.haml (hunk 4 above)
Comment (Gnito) [inline app/views/layouts/react_page.haml]: MikkoP, but reading from the original commit, it looks pretty clear that this is just graceful deprication (for OS, I guess). I'll remove

**Pull request merged**